README.md

```diff
@@ -150,3 +150,17 @@ In general you can pipe any endless stream to fblog.
 ```bash
 fblog --generate-completions <shell>
 ```
+
+
+## configuration file
+
+fblog load a `fblog.toml` file.
```

Comment:
[nit] Maybe more like

`fblog` reads its configuration from a file called `fblog.toml`, which is located:
* Linux: ...
* Windows: ...
* MacOS: ...